plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestGlobalTransactionMySqlConnectorTest.java

```diff
@@ -17,18 +17,19 @@
 
 import static io.trino.plugin.mysql.MySqlQueryRunner.createMySqlQueryRunner;
 import static io.trino.tpch.TpchTable.CUSTOMER;
+import static io.trino.tpch.TpchTable.LINE_ITEM;
 import static io.trino.tpch.TpchTable.NATION;
 import static io.trino.tpch.TpchTable.ORDERS;
 import static io.trino.tpch.TpchTable.REGION;
 
-public class TestGlobalTransactionMySqlIntegrationSmokeTest
-        extends BaseMySqlIntegrationSmokeTest
+public class TestGlobalTransactionMySqlConnectorTest
+        extends BaseMySqlConnectorTest
```

Comment:
```
// TODO define shorter tests set that exercises various read and write scenarios (a.k.a. "a smoke test")
```

and see @kokosing 's comment https://github.com/trinodb/trino/pull/6986#discussion_r580514339